zerver/tests/fixtures/markdown_test_cases.json

```diff
@@ -932,6 +932,121 @@
       "input": "<h1>*<h1>[<h2>Static types in Python</h2>](https://blog.zulip.com/2016/10/13/static-types-in-python-oh-mypy)</h1>*</h1>",
       "expected_output": "<p>&lt;h1&gt;<em>&lt;h1&gt;<a href=\"https://blog.zulip.com/2016/10/13/static-types-in-python-oh-mypy\">&lt;h2&gt;Static types in Python&lt;/h2&gt;</a>&lt;/h1&gt;</em>&lt;/h1&gt;</p>",
       "marked_expected_output": "<p>&lt;h1&gt;<em>&lt;h1&gt;<a href=\"https://blog.zulip.com/2016/10/13/static-types-in-python-oh-mypy\">&lt;h2&gt;Static types in Python&lt;/h2&gt;</a>&lt;/h1&gt;</em>&lt;/h1&gt;\n\n</p>"
+    },
+    {
+      "name": "auto_shorten_links",
+      "input": "https://github.com/zulip/zulip",
+      "expected_output": "<p><a href=\"https://github.com/zulip/zulip\">https://github.com/zulip/zulip</a></p>"
+    },
+    {
+      "name": "auto_shorten_issues_links",
+      "input": "https://github.com/zulip/zulip/issues/11895",
+      "expected_output": "<p><a href=\"https://github.com/zulip/zulip/issues/11895\">zulip/zulip#11895</a></p>"
+    },
+    {
+      "name": "auto_shorten_pull_links",
+      "input": "https://github.com/zulip/zulip/pull/16665",
+      "expected_output": "<p><a href=\"https://github.com/zulip/zulip/pull/16665\">zulip/zulip#16665</a></p>"
```

Comment:
Can you change these tests to look at repositories that are not zulip/zulip?  The organization name matching the project name could hide bugs.  `zulip/zulip-mobile` would be a better choice.